package/src/components/ImageGallery/ImageGallery.tsx

```diff
@@ -187,27 +186,13 @@ export const ImageGallery = <
   const [currentBottomSheetIndex, setCurrentBottomSheetIndex] = useState(0);
   const animatedBottomSheetIndex = useSharedValue(0);
 
-  /**
-   * Fade animation for screen, it is always rendered with pointerEvents
-   * set to none for fast opening
-   */
-  const showScreen = useSharedValue(0);
-  const fadeScreen = (show: boolean) => {
-    'worklet';
-    showScreen.value = withTiming(show ? 1 : 0, {
-      duration: 250,
-      easing: Easing.out(Easing.ease),
-    });
-  };
-
   /**
    * Run the fade animation on visible change
    */
   useEffect(() => {
     if (visible) {
       Keyboard.dismiss();
     }
-    fadeScreen(visible);
```

Comment:
Could you also check if prop `visible` is required anymore. Same for `MessageOverlay` component.

Reply:
Umm, they can be removed. I will have a look at them now. Thanks 😄